### DISCUSSION TITLE
Unable to RUN make dist for rabbitmq_auth_backend_ldap.

### DISCUSSION THREAD

**Discussion by ramaraju250** [rabbitmq/rabbitmq-server#2744] [Other] [answered]
**To Reproduce:**

git clone https://github.com/rabbitmq/rabbitmq-server.git
cd rabbitmq-server/deps/rabbitmq_auth_backend_ldap/
make dist

**The make dist command is failing with below error**
 
```
GEN    escript/rabbitmqctl
{"init terminating in do_boot",{badarg,[{erlang,apply,[{'Elixir.Kernel.CLI.Config',[],<<".">>,[],true,[],[],false},update_commands,[#Fun<Elixir.Kernel.CLI.18.41400255>]],[]},{'Elixir.Kernel.CLI',process_argv,2,[{file,"lib/kernel/cli.ex"},{line,191}]},{'Elixir.Kernel.CLI',main,1,[{file,"lib/kernel/cli.ex"},{line,14}]},{init,start_em,1,[]},{init,do_boot,3,[]}]}}
init terminating in do_boot ({badarg,[{erlang,apply,[{_},update_commands,[_]],[]},{Elixir.Kernel.CLI,process_argv,2,[{_},{_}]},{Elixir.Kernel.CLI,main,1,[{_},{_}]},{init,start_em,1,[]},{init,do_boot,3

Crash dump is being written to: erl_crash.dump...done
make[2]: *** [Makefile:108: escript/rabbitmqctl] Error 1
make[2]: Leaving directory '/root/RMQ3.8.10/rabbitmq-server/deps/rabbitmq_cli'
make[1]: *** [erlang.mk:4512: deps] Error 2
make[1]: Leaving directory '/root/RMQ3.8.10/rabbitmq-server/deps/rabbit'
make: *** [erlang.mk:4512: deps] Error 2
```

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
The exception comes from the Elixir standard library. Make sure you have a supported Elixir version installed. There is no reason not to use 1.10 or 1.11 on Erlang 22 or 23.

**Reply by ramaraju250** [reply]
Thanks, it resolved after updating the elixir to 1.10
